pkg/registry/authorization/selfsubjectrulesreview/rest.go

```diff
@@ -55,7 +55,7 @@ func (r *REST) Destroy() {
 	// here explicitly.
 }
 
-// Create attempts to get self subject rules in specific namespace.
+// Create returns attributes of the subject making the request.
```

Comment:
why did this change?

Reply:
Probably a mistake. I will revert this one.

Reply:
Reverted.